reference/generators/pkg_config.rst

```diff
@@ -5,7 +5,7 @@ pkg_config
 ==========
 
 Generates pkg-config files named *<PKG-NAME>.pc* (where ``<PKG-NAME`` is the name declared by dependencies in
-``cpp_info.name`` or in ``cpp_info.names["pkg_config"]`` if specified), containing a
+``cpp_info.names["pkg_config"]`` if specified), containing a
```

Comment:
also `set_property("pkg_config_name")`, otherwise, `PkgConfigDeps` is not supported

Reply:
I agree, as properties are supported by this generator it would be good to add a note about set_property here